src/olympia/conf/prod/settings.py

```diff
@@ -85,34 +85,22 @@
     'default': {
         'client_id': env('FXA_CLIENT_ID'),
         'client_secret': env('FXA_CLIENT_SECRET'),
-        'content_host': 'https://accounts.firefox.com',
-        'oauth_host': 'https://oauth.accounts.firefox.com/v1',
-        'profile_host': 'https://profile.accounts.firefox.com/v1',
-        'redirect_url':
-            'https://%s/api/v3/accounts/authenticate/' % DOMAIN,
-        'scope': 'profile',
+        # redirect_url' = 'https://%s/api/v3/accounts/authenticate/' % DOMAIN,
     },
     'amo': {
         'client_id': env('AMO_FXA_CLIENT_ID'),
         'client_secret': env('AMO_FXA_CLIENT_SECRET'),
-        'content_host': 'https://accounts.firefox.com',
-        'oauth_host': 'https://oauth.accounts.firefox.com/v1',
-        'profile_host': 'https://profile.accounts.firefox.com/v1',
-        'redirect_url': 'https://addons.mozilla.org/api/v3/accounts/authenticate/?config=amo', # noqa
-        'scope': 'profile',
-        'skip_register_redirect': True,
+        # redirect_url = 'https://addons.mozilla.org/api/v3/accounts/authenticate/?config=amo', # noqa
```

Comment:
Hmm, if you're commenting this one out in all the configs it's probably best to add an additional comment why it's commented out. Are you using this for documentation reasons?

Reply:
There's no way of knowing what the redirect url is going to if you didn't personally create the fxa client, so I'm documenting it, yes.

Reply:
Okay, then it'd make sense to add some more context as part of that documentation, e.g the purpose of that line and why it's here etc.

Reply:
https://github.com/mozilla/addons-server/pull/12532/commits/8a5b67d43e4b41d1ffaf73648c01d0a2ddb58367